software/o_c_REV/OC_ui.h

```diff
@@ -25,7 +25,7 @@ enum UiControl {
   CONTROL_ENCODER_L   = 0x20,
   CONTROL_ENCODER_R   = 0x30,
```

Comment:
```suggestion
  CONTROL_ENCODER_R   = 0x40,
```
Since these control ids are intended to be combinable in a bitmask. 
IRC only the buttons are ever used in that manner so it probably doesn't matter (until it does...)

Reply:
That's one of Max's changes, but your version looks reasonable to me.